src/Http/Http/src/StreamPipeReader.cs

```diff
@@ -227,7 +227,7 @@ public override async ValueTask<ReadResult> ReadAsync(CancellationToken cancella
                 try
                 {
                     AllocateReadTail();
-#if NETCOREAPP2_2
+#if NETCOREAPP3_0
                     var length = await _readingStream.ReadAsync(_readTail.AvailableMemory.Slice(_readTail.End), tokenSource.Token);
 #elif NETSTANDARD2_0
```

Comment:
`#if NETSTANDARD2_0` can be removed now, right?

Reply:
We're keeping it because this code is going to get moved to corefx and support netstandard.